downstream/modules/platform/proc-controller-create-container-group.adoc

```diff
@@ -0,0 +1,63 @@
+[id="controller-create-container-group"]
+
+= Creating a container group
+
+A `ContainerGroup` is a type of `InstanceGroup` that has an associated credential that enables you to connect to an OpenShift cluster. 
+
+.Prerequisites
+
+* A namespace that you can launch into.
+Every cluster has a "default" namespace, but you can use a specific namespace.
+* A service account that has the roles that enable it to launch and manage pods in this namespace.
+* If you are using {ExecEnvShort}s in a private registry, and have a container registry credential associated with them in {ControllerName}, the service account also needs the roles to get, create, and delete secrets in the namespace. 
+If you do not want to give these roles to the service account, you can pre-create the `ImagePullSecrets` and specify them on the pod spec for the `ContainerGroup`. 
+In this case, the {ExecEnvShort} should not have a container registry credential associated, or {ControllerName} attempts to create the secret for you in the namespace.
+* A token associated with that service account. 
+An OpenShift or Kubernetes Bearer Token.
+* A CA certificate associated with the cluster.
+
+The following procedure explains how to create a service account in an Openshift cluster or Kubernetes, in order to be used to run jobs in a container group through {ControllerName}. 
```

Comment:
I think there's another instance of Openshift that could be replaced with OpenShift